virtcontainers/fc.go

```diff
@@ -491,6 +492,24 @@ func (fc *firecracker) client() *client.Firecracker {
 	return fc.connection
 }
 
+func (fc *firecracker) createJailedDrive(name string) (string, error) {
```

Comment:
Could you add some unit tests for this to ensure the path is "jailed" if `fc.jailed == true`?